core/cmd/wandb-core/main.go

```diff
@@ -10,6 +10,7 @@ import (
 	"runtime/trace"
 
 	"github.com/getsentry/sentry-go"
+	// "github.com/wandb/wandb/core/internal/processlib"
```

Comment:
let's remove this?

Reply:
i know it is related to the commented out code, but it just looks like dead code

Reply:
moved to be a command line argument and uncommented the code